examples/example.conf

```diff
@@ -5,16 +5,16 @@
 time_step = 60 
 
 # Forcings begin here (year/month/day:hour) (hour optional)
-start = 1950/1/1:0  
+start = 1949/1/1:0  
 
 # Forcings end at this date (year/month/day)
-stop = 1950/1/31
+stop = 1949/12/31
 
 # Input and output directories
-forcing = ./examples/forcing_sub.nc
-domain  = ./examples/domain_sub.nc
+forcing = ./examples/forcing
```

Comment:
That may make it clearer.  This was a carryover from the example VIC data I have been using, which didn't have an extension.